andela/ah-the-jedi-frontend
# #166571933 Fix app responsiveness and social login styling

### What does this PR do?
Fix app responsiveness and stlye social buttons

### Description of Task(s) to be completed?
- Add colors to variables.css
- Fix styling of social login buttons on some browsers
- Fix responsiveness of signup page
- Fix responsiveness of login page

### How should this be manually tested?

Clone this repository on local machine and CD into project folder
`$ git clone git@github.com:andela/ah-the-jedi-frontend.git`
`$ cd ah-the-jedi-frontend`

Check out to the PR's branch
`$ git checkout bg-fix-app-responsiveness-166571933`

install dependencies and run the dev server
`npm install`
`npm run start:dev`

navigate to login and signup page and change sizes to view responsiveness

### What are the relevant PT stories?
[#166571933](https://www.pivotaltracker.com/story/show/166571933)

### Screenshots
#### Sign Up
![image](https://user-images.githubusercontent.com/11965186/59260460-513b0d00-8c44-11e9-9787-3167c3e2da4f.png)

#### Sign In
![image](https://user-images.githubusercontent.com/11965186/59260573-88a9b980-8c44-11e9-856e-d475cc3e6582.png)
